app/src/main/java/com/kickstarter/viewmodels/DiscoveryViewModel.java

```diff
@@ -214,7 +214,7 @@ public DiscoveryViewModel(final @NonNull Environment environment) {
       .take(1)
       .map(Intent::getAction)
       .filter(Intent.ACTION_MAIN::equals)
-      .map(__ -> DiscoveryParams.builder().staffPicks(true).build());
+      .map(__ -> DiscoveryParams.builder().build());
```

Comment:
Okay. I'm not sure how crucial it is to the tests, but throughout `DiscoveryFragmentViewModelTest` we set staff picks to true when loading the initial params, e.g.: https://github.com/kickstarter/android-oss/pull/31/files#diff-295515ebc9274113707ed3b807dd41dcR137. Seems like the new default would not have that param set to true.

Would recommend doing a pass throughout the codebase where `staffPicks` is set to true and make sure we don't have any outdated assumptions about what the initial load looks like.